apps/dashboard/src/lib/features/onboarding/components/verify-email-modal.svelte

```
@@ -6,6 +6,7 @@
   import { currentOrg } from '$lib/utils/store/org';
   import { profile } from '$lib/utils/store/user';
   import { authClient } from '$lib/utils/services/auth/client';
+  import { logout } from '$lib/utils/functions/logout';
```

ESLint

[error] 'logout' is defined but never used.

(unused-imports/no-unused-imports)